platform/core/src/services/MeasurementService/MeasurementService.js

```diff
@@ -524,19 +560,25 @@ class MeasurementService {
     };
 
     if (this.measurements[internalUID]) {
+      newMeasurement.selected = this.measurements[internalUID].selected;
```

Comment:
can you please add a comment above that you are doing this since there is no `isSelected` state on the annotation, and that actually is the correct way of doing this?

Reply:
certainly